### DISCUSSION TITLE
Crash with Optifabric?

### DISCUSSION THREAD

**Discussion by Yis43455** [gnembon/fabric-carpet#931] [Q&A]
Hi;
I need help or information... in my folder mods, I have only "Mod menu, Fabric API, Optifabric, Optifine G9 pre 21 and Fabric carpet"
![mods](https://user-images.githubusercontent.com/86010741/122251009-c246d800-ce8f-11eb-84a2-a7896c3a0130.PNG)
But my Minecraft launcher crash every time, previous to the loading screen.
My question is: The Fabric carpet need any extension or am I a missing mod? The Fabric carpet isn't compatible with Optifine 1.17
Thank you.
PD: I'm new in fabric or Minecraft TEC and my PC isn't very good I need Optifine to play "good".

**Comment by altrisi** [comment]
Yes, it's known that Optifabric in 1.17 is not compatible with Carpet. The currently tracking issue is #923 in Carpet and Chocohead/Optifabric#245 in Optifabric's side.

Hopefully Sodium comes soon to 1.17 and we can drop OF once and for all.
